packages/components/src/__alpha__/SingleSelect/subcomponents/List/List.tsx

```diff
@@ -1,18 +1,28 @@
-import React, { type PropsWithChildren } from 'react'
-import classNames from 'classnames'
-import { ListBox as RACListBox, type ListBoxProps } from 'react-aria-components'
-import { type SelectItem, type SelectSection } from '../../types'
+import React from 'react'
+import { useListBox } from 'react-aria'
+import { type ListProps, type SelectItem } from '../../types'
+import { ListItem } from '../ListItem'
+import { ListSection } from '../ListSection'
 import styles from './List.module.css'
 
-export const List = ({
-  children,
-  className,
-  ...props
-}: ListBoxProps<SelectItem | SelectSection> & PropsWithChildren): React.ReactElement => {
+export const List = <T extends SelectItem>({
+  state,
+  listBoxOptions,
+  listBoxRef,
+}: ListProps<T>): JSX.Element => {
+  const { listBoxProps } = useListBox(listBoxOptions, state, listBoxRef)
+
+  const renderNode = (node: any): JSX.Element => {
+    if (node.type === 'section') {
+      return node.rendered && <ListSection section={node} state={state} />
+    } else {
+      return <ListItem key={node.key} item={node} state={state} />
+    }
+  }
+
   return (
-    <RACListBox className={classNames(styles.list, className)} {...props}>
-      {children}
-    </RACListBox>
+    <ul {...listBoxProps} ref={listBoxRef} className={styles.list}>
```

Comment:
Was doing another once over in VoiceOver and noticed that the bullet list was being announced - I think this might have been contributing to some of the strange behaviors with VO on Friday.

`list-style: none` is one of the few CSS properties that can impact the accessible tree so would recommend adding that in. Would recommend adding that in and spinning up VO